nautilus_trader/test_kit/strategies/tester_data.py

```diff
@@ -54,7 +54,7 @@ class DataTesterConfig(ActorConfig, frozen=True):
     request_instruments: bool = False
     request_quotes: bool = False
     request_trades: bool = False
-    request_bars: bool = False
+    request_bars: bool = True
```

Comment:
Lets keep this set to `False` as the default tester config should be somewhat of a "clean slate".

Reply:
done, thx